src/mutations.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+from genepy.google import gcp
+import numpy as np
+from collections import Counter
+
+def download_maf_from_workspace(refwm, sample_set_ids = ['all_ice', 'all_agilent'],
```

Comment:
this is probably not needed. I didn't realize about the 'all' sample_set when I made this.

Reply:
We can always remove it later :)